### DISCUSSION TITLE
直播界面弹幕窗口位置调整

### DISCUSSION THREAD

**Discussion by killlller** [the1812/Bilibili-Evolved#1488] [功能建议 / Ideas]
希望可以有像YouTube一样的窗口全屏（Theater mode). 弹幕窗口移动到整体的下方，利于竖屏用户观看，谢谢。

**Comment by Geeyun-JY3** [comment]
有个 `直播全屏弹幕栏` 功能不知道能不能解决你的部分需求

**Reply by mzmmzm** [reply]
我觉得他指的是弹幕显示栏而不是`直播全屏弹幕栏`提供的发送弹幕栏

**Reply by Geeyun-JY3** [reply]
当时没理解，现在明白了。

**Comment by chenxi-Eumenides** [comment]
用竖屏摸鱼时感受到了一点不便，希望有大佬能开发一个QAQ

**Reply by chenxi-Eumenides** [reply]
#4179 我把我的想法写的更详细了一点。
